iohub/ngff/nodes.py

```diff
@@ -193,7 +193,7 @@ def __delitem__(self, key):
 
     def __contains__(self, key):
         key = normalize_storage_path(key)
-        return key in self._member_names
+        return key.lower() in [name.lower() for name in self._member_names]
```

Comment:
I consider this a bug if a `key in node` is `True` does not guarantee that `node[key]` succeeds on all platforms.